docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,208 @@
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in [DNS Query and
+Response Defaults]. The handling of the DNS responses on the DNS queries follow,
+unless otherwise specified below, what is specified for [DNS Response] in the
+same specification.
+
+1. Create a [DNS Query] with query type TXT and query name *Child Zone* ("TXT
+   query").
+
+2. Create an empty set of pairs of IP addresses and strings, "SPF-Policies".
+
+3. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+4. For each name server in *Name Server IP* do:
+
+   1. Send *TXT Query* to the name server and collect the [DNS Response].
+
+   2. Go to the next name server IP address if at least one of the following
+      criteria is met:
+
+      1. There is no DNS response.
+      2. [RCODE Name] of the response is not "NoError".
+      3. The AA flag is not set in the response.
+
+   3. If the name server responds with no TXT record, then add the pair
+      consisting of the *Name Server IP* and the empty string to the
+      *SPF-Policies* set.
+
+   4. If the name server responds with at least one TXT record and none is an
+      [SPF TXT record], then add the pair consisting of the *Name Server IP*
+      and the empty string to the *SPF-Policies* set.
+
+   5. If the name server responds with at least one TXT record that is an [SPF
+      TXT record], then, for each [SPF TXT record] do:
+
+       1. [Concatenate] all strings in the RDATA field.
+       2. Lowercase the resulting string.
+       3. Add a pair consisting of the *Name Server IP* and the lowercase
+          string thus derived from the RDATA field to the *SPF-Policies* set.
+
+   6. Go to the next name server.
+
+5. If the *SPF-Policies* set is empty, then output
+   *[Z11_UNABLE_TO_CHECK_FOR_SPF]* and terminate the test.
+
+6. If all the pairs in the *SPF-Policies* set contain empty strings, then
+   output *[Z11_NO_SPF_FOUND]* and terminate the test.
+
+7. Compare the set of *SPF-Policies* retrieved from all name servers. If at
+   least two different name servers have returned different sets of SPF
+   policies, then:
+
+   1. Output *[Z11_INCONSISTENT_SPF_POLICIES]*.
+   2. Group *SPF-Policies* by equal sets of SPF policies, such that a set of
+      SPF policies is mapped to the list of *Name Server IPs* that returned it.
+   3. For each such group of name servers, output
+      *[Z11_DIFFERENT_SPF_POLICIES_FOUND]*.
+   4. Terminate the test.
+
+8. If the *SPF-Policies* set contains at least two pairs with the same IP
+   address, then output *[Z11_SPF1_MULTIPLE_RECORDS]* with the list of
```

Comment:
I think it would be easier to read if it says "... two entries with the same IP address ...".

Reply:
Fixing.